composeApp/src/ooniMain/composeResources/values-ar/strings-organization.xml

```diff
@@ -70,5 +70,5 @@
   <string name="Settings_About_Content_Blog">مدونة</string>
   <string name="Settings_About_Content_Reports">التقارير</string>
   <string name="Settings_About_Content_DataPolicy">سياسة OONI في البيانات   </string>
-    <string name="Dashboard_Runv2_Overview_Uninstall_Prompt">You will be able to install this link again only from the original link sent by the creator. Measurements from this link will be deleted, but will be accessible from explorer.</string>
-</resources>
+  <string name="Dashboard_Runv2_Overview_Uninstall_Prompt">You will be able to install this link again only from the original link sent by the creator.</string>
+</resources>
```

Comment:
We shouldn't be rolling back this.

Reply:
😄  we need to merge the translations spreadsheets.